include/CppUTest/UtestMacros.h

```diff
@@ -249,8 +249,10 @@
 #define BITS_EQUAL_TEXT(expected, actual, mask, text)\
   BITS_LOCATION(expected, actual, mask, text, __FILE__, __LINE__)
 
+#define ROBUST_SIZEOF(type) (2*sizeof(type)/sizeof(short)) /* For the Cl2000, sizeof(char) == sizeof(short) == 1 */
```

Comment:
You have a point, of course, but in this particular case, I object to #ifdefing the test case, because for the 16 bytes platform, the result displayed will be WRONG. So we do need to fix the display. If you have a cleaner idea than the macro, I'm willing to implement that.